lib/libzutil/zutil_import.c

```diff
@@ -1006,35 +1006,39 @@ zpool_read_label(int fd, nvlist_t **config, int *num_labels)
 
 	if (lio_listio(LIO_WAIT, aiocbps, VDEV_LABELS, NULL) != 0) {
 		int saved_errno = errno;
-		int tmpret = -1;
+		errno = 0;
+		int do_slow = 0;
```

Comment:
this would probably prefer to be a `boolean_t`

Reply:
As you've already noticed, done.